frameworks/cassandra/tests/test_racks.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import pytest
+import sdk_install
+import sdk_tasks
+from tests import config
+from tests import nodetool
+
+log = logging.getLogger(__name__)
+
+@pytest.fixture(scope='module', autouse=True)
+def configure_package(configure_security):
+    try:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+        yield # let the test session execute
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+@pytest.mark.sanity
+def test_rack():
+    sdk_install.install(config.PACKAGE_NAME, config.SERVICE_NAME, 3)
+
+    # dcos task exec node-0-server bash -c 'JAVA_HOME=jre1.8.0_144 apache-cassandra-3.0.14/bin/nodetool status'
+    raw_status = sdk_tasks.task_exec('node-0-server', "bash -c 'JAVA_HOME=jre1.8.0_144 apache-cassandra-3.0.14/bin/nodetool status'")
```

Comment:
These hard coded values are likely to get stale quickly.

Reply:
Affirmative

Reply:
`JAVA_HOME=$(ls -d $MESOS_SANDBOX/jre*/)`

Reply:
Can you put the CASSANDRA_VERSION into the task env?

Reply:
Or something like: `CASSANDRA_VERSION = sdk_marathon.get_config(foldered_name)['env']['CASSANDRA_VERSION']`

Reply:
You can just use asterisks. Did it in this PR: https://github.com/mesosphere/dcos-commons/pull/1809/files#diff-fa78f986c873acfad32eed0038c64638R149